site/docs/v1/tech/themes/kickstart-custom-theme.adoc

```diff
@@ -0,0 +1,400 @@
+
+To manually set the theme of the default tenant, navigate to "Tenants" and hit the "edit" button. In the "Theme" panel, select "Dark Theme" and click "Save".
+
+image::themes/kickstart-theme/select-dark-theme.png[Set the theme,width=1200]
+
+== Running the Application
+
+Now that everything is set up and our theme has been applied, we can run the application. Navigate to the `fusionauth-example-5-minute-guide` and use `npm` to start the application.
+
+```bash
+npm install
+npm start
+```
+
+Open an incognito window and visit `http://localhost:3000`.
+
+You will be taken to the same landing page that you saw in the 5-minute guide. However, when you click "Login" this time, you will see your custom theme applied to the OAuth authorize page.
+
+image::themes/kickstart-theme/dark-authorize.png[Dark theme applied,width=1200, role=bottom-cropped]
+
+Enter the same credentials you used to login to the admin panel and click "Logout" to see the OAuth logout page.
+
+
+== Modifying the Default Messages
+
+Let's take it one step further and assume we want to change the content of some of the messages on the OAuth pages. For example, consider the "Forgot your password?" message, which shows up on the OAuth authorize page.
+
+image::themes/kickstart-theme/forgot-password.png[Forgot password,width=1200, role=bottom-cropped]
+
+Let's say we want to change this to say "Forgot your password? Click here." We can do this by adding a `defaultMessages` property to `json/updateTheme.json`.
+
+The `defaultMessages` string requires at least all of the messages defined in the FusionAuth default shipped messages file to be present, as it updates all messages as a single unit. The easiest way to accomplish this is to create a new file called `defaultMessages.txt` in your `kickstart` folder and copy-paste these messages into it.
+
+The messages can be accessed by editing your custom theme, navigating to the "Messages" page, and clicking the "Edit" button.
+
+image::themes/kickstart-theme/messages.png[Theme messages,width=1200]
+
+Copy the entire contents of that box into your `defaultMessages.txt` file, find the `forgot-your-password` message (line 65), and modify it to "Forgot your password? Click here."
+
+```json
+{
+  "method" : "PATCH",
+  "url" : "api/theme/#{themeID}",
+  "body" : {
+    "theme" : {
+      "stylesheet" : "@{darkTheme.css}",
+      "defaultMessages" : "@{defaultMessages.txt}"
+    }
+  }
+}
+````
+```
+[NOTE]
+====
+Kickstart will not run if it sees any users, API keys, or applications in the FusionAuth database. This is to prevent data loss. If you can login to the FusionAuth administrative user interface, Kickstart will not run.
+====
+
+Once you have modified `updateTheme.json`, you will need to clear the volumes created when you launched the FusionAuth instance to allow the Kickstart to rerun. When you relaunch the instance using the `docker-compose up` command and apply the dark theme, the result will be as below with the updated message:
```

Comment:
Would be great to show people how to do this:

```
docker compose down -v
```

And mention that this will totally destroy all data stored in the instance, so only useful for dev.